eng/emitter-package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "main": "dist/src/index.js",
+    "dependencies": {
+      "@azure-tools/typespec-go": "D:/Go/src/github.com/Azure/autorest.go/packages/typespec-go"
+    },
+    "devDependencies": {
+      "@typespec/compiler": "0.57.0",
+      "@typespec/http": "0.57.0",
+      "@typespec/openapi": "0.57.0",
+      "@typespec/rest": "0.57.0",
+      "@typespec/versioning": "0.57.0",
+      "@azure-tools/typespec-azure-core": "0.43.0",
```

Comment:
Can you add this dependency `@azure-tools/typespec-autorest": "0.43.0` to avoid failure when service's typespec has this dependency?